src/lib/coda_networking/coda_networking.ml

```diff
@@ -869,6 +930,16 @@ let create (config : Config.t)
         in
         if valid_protocol_versions then result else None
   in
+  let get_some_initial_peers_rpc conn ~version:_ () =
+    [%log debug] "Sending some initial peers to peer with IP %s"
+      (Unix.Inet_addr.to_string conn.Peer.host) ;
```

Comment:
Would it be useful to log the peer id, in addition to the host? The IP doesn't identify the node, necessarily.

there are other RPC logs which print just the IP, maybe it's useful to add them there, too